contrib/nodejs/globals.js

```diff
@@ -104,10 +104,112 @@ function clearImmediate(id) {}
 function Process() {}
 
 /**
+ * @see http://nodejs.org/api/process.html#process_process_abort
+ */
+Process.prototype.abort = function () {};
```

Comment:
If I remember correctly, this just tells the compiler that abort is a function, but the compiler won't know its param or return types. You need to add `@return {void}` to get the compiler to know that.